components/portal/messages/filters.js

```diff
@@ -68,6 +68,31 @@ define(['angular'], function(angular) {
         });
       };
     })
+    .filter('filterMessagesWithInvalidDates', function() {
+      return function(messages) {
+          var thePresent = moment.now();
+          return messages.filter(function(message) {
+            var goDate = moment(message.goLiveDate);
+            var stopDate = moment(message.expireDate);
+            // If neither date is populated, message is valid.
+            if (!goDate.isValid() && !stopDate.isValid()) {
+              return message;
+            }
+            if (goDate.isValid() && stopDate.isValid()) {
+              if (goDate.isBefore(thePresent)
```

Comment:
Moment has the [isBetween()](https://momentjs.com/docs/#/query/is-between/) function for this kind of check. Makes no difference what you use, but it's out there!